resources/assets/js/videos/components/videoScreen/annotationPlayback.vue

```diff
@@ -128,11 +128,12 @@ export default {
         refreshSingleAnnotation(annotation) {
             let source = this.annotationSource;
 
-            let newFeature = this.createFeature(annotation);
-            let oldFeature = source.getFeatureById(annotation.id)
-            
-            source.removeFeature(oldFeature);
-            source.addFeature(newFeature);
+            let feature = source.getFeatureById(annotation.id)
+
+            source.removeFeature(feature);
+
+            feature.set('color', annotation.labels[0].label.color);
+            source.addFeature(feature);
```

Comment:
Looks good! Is the call to `removeFeature()` and `addFeature()` even necessary now?